### PR TITLE
feat: filter functionality added

### PR DESCRIPTION
**Pull request by vikrantkode**
# Open for Review

## Clear Filters

### CODE REVIEW

**Review by Asuraking1n — APPROVED** [review]
UI looks great, 
You imported image that was cool,many people are not doing it.
Looks good to merge

File: src/components/Cart/Cart.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    grid-area: main;
+}
+
+.btn_container{
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+}
+
+/* .btn{
```
Comment (harshitpaliwal95) [inline src/components/Cart/Cart.css:25]: remove commented code

File: src/reducer/reducerFunction.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    }
+    case "SORT_BY_RATING": {
+      return { ...state, sortRating: action.payload };
+    }
+    case "SUNGLASSES": {
+      return {
+        ...state,
+        sortCategory: {
+          ...state.sortCategory,
+          bySunglasses: !state.sortCategory.bySunglasses,
```
Comment (harshitpaliwal95) [inline src/reducer/reducerFunction.jsx:17]: you can use variable for SORT_BY_PRICE and for other cases

**Review by harshitpaliwal95 — APPROVED** [review]
- code looks good you can merge

File: src/reducer/reducerFunction.jsx (hunk 2 above)
Comment (vikrantkode) [reply src/reducer/reducerFunction.jsx:17]: It didnt clicked for me....will do the change after finishing projects.

File: src/components/Cart/Cart.css (hunk 1 above)
Comment (vikrantkode) [reply src/components/Cart/Cart.css:25]: yeah will do that

**Pull request merged**